### PR TITLE
fix for #164

### PR DESCRIPTION
**Pull request by atharva-upadhye-01**
## Issue that this pull request solves
[Frontend] Make Get Involved page

 Closes: #164

## Proposed changes
Added get involved page

### Brief description of what is fixed or changed
Added route `/get-involved` and made changes to footer accordingly
Created getInvolved page in its respective folder structure and called that component in footer

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
image

### CODE REVIEW

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @atharva-upadhye-01 please change the card color to `#016795` this also the font-family of the heading should be same as other page like home page

**Comment by atharva-upadhye-01** [comment]
@Kajol-Kumari  Can you please add DWOC ?
I have enrolled in it.

File: frontend/src/components/involvement/get_involved.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import './get_involved.css';
+// import icon from
```
Comment (Kajol-Kumari) [inline frontend/src/components/involvement/get_involved.js:3]: please remove this line

File: frontend/src/components/involvement/get_involved.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+    img {
+        max-width: 100vw;
+    }
+
+    .card-item,.card-content {
+        width: 100%;
+        height: 100%;
+    }
+}
```
Comment (Kajol-Kumari) [inline frontend/src/components/involvement/get_involved.css:94]: leave one line space at the end of file

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @atharva-upadhye-01 thanks for incorporating the suggested changes please make the minor change sand fix the heading fir small screen and it will be good to go :)
Screenshot from 2021-01-07 19-31-19

File: frontend/src/components/involvement/get_involved.css [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+#title {
+    display: flex;
+    align-items: center;
+    font-size: 4em;
+    font-weight: bold;
+    height: 1em;
+    margin-top: 4rem;
+    font-family: Lato,'Helvetica Neue',Arial,Helvetica,sans-serif;
```
Comment (Surajbokde) [inline frontend/src/components/involvement/get_involved.css:29]: font-family: "Futura LT Book", sans-serif;

make this change

**Review by Surajbokde — CHANGES_REQUESTED** [review]
@atharva-upadhye-01 
change card background color to **#121f3a;**
and on hovermake it   gradient color
**background: rgb(255, 0, 90);
  background: linear-gradient(
    45deg,
    rgba(255, 0, 90, 1) 0%,
    rgba(10, 24, 61, 1) 90%
  );**

**Comment by Kajol-Kumari** [comment]
> @atharva-upadhye-01
> change card background color to **#121f3a;**
> and on hovermake it gradient color
> **background: rgb(255, 0, 90); background: linear-gradient( 45deg, rgba(255, 0, 90, 1) 0%, rgba(10, 24, 61, 1) 90% );**

Hey @Surajbokde  believe the color combination is looking preety good so let's stick with the one that is already implemented in this PR :)

\cc: @atharva-upadhye-01

File: frontend/src/components/involvement/get_involved.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  ]
+  return (
+    <main>
+      <div id="hero">
+        <div id="heading">
+          <h1 id="title">GET INVOLVED!</h1>
+        </div>
+      </div>
+      <div id="allCards">
+        {arrayCards.map(element => {
```
Comment (chait04) [inline frontend/src/components/involvement/get_involved.js:33]: provide key to the div.. put ( i ) as a argument in map function and just pass i as key to div

File: frontend/src/components/involvement/get_involved.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    <main>
+      <div id="hero">
+        <div id="heading">
+          <div id="title">GET INVOLVED!</div>
+        </div>
+      </div>
+      <div id="allCards">
+        {arrayCards.map(element => {
+          return (
+            <div className="card-item">
```
Comment (mohit355) [inline frontend/src/components/involvement/get_involved.js:35]: Pass a key parameter to this div.
Use the 2nd parameter of the map function as a key for the first div that mounted all other tags inside the map function.
ex: -  key={ index}    'index;   is the 2nd parameter.

**Review by mohit355 — CHANGES_REQUESTED** [review]

**Comment by atharva-upadhye-01** [comment]
Let me know suggestions if any :)

**Review by Kajol-Kumari — APPROVED** [review]
Hey @atharva-upadhye-01 thanks for incorporating the suggested changes. It looks good to me :)

**Pull request merged**